src/OmniSharp.DotNetTest/Helpers/TestFeaturesDiscover.cs

```diff
@@ -59,5 +75,24 @@ private static bool IsDerivedFromFactAttribute(ITypeSymbol symbol)
 
             return false;
         }
+
+        private bool IsNUnitTest(ITypeSymbol symbol)
+        {
+            string fullName;
+            do
+            {
+                fullName = $"{symbol.ContainingNamespace}.{symbol.Name}";
+                if (fullName == "NUnit.Framework.TestAttribute"
+                    || fullName == "NUnit.Framework.TestCaseAttribute"
+                    || fullName == "NUnit.Framework.TestCaseSourceAttribute")
+                {
+                    return true;
+                }
+
+                symbol = symbol.BaseType;
+            } while (symbol.Name != "Object");
```

Comment:
The changes in the VS Code plugin and in Omnisharp-Roslyn work independently. The new Omnisharp-Roslyn will work with the old plugin, and the new plugin will work with the "old" Omnisharp-Roslyn. In both cases xUnit testing still works.